code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -789,6 +703,7 @@
 	var/new_freq = get_pin_data(IC_INPUT, 1)
 	var/new_code = get_pin_data(IC_INPUT, 2)
 	if(isnum_safe(new_freq) && new_freq > 0)
+		new_freq = Clamp(RADIO_LOW_FREQ, new_freq, RADIO_HIGH_FREQ)
```

Comment:
может between?

Reply:
"Clamps" a number to a given allowable range from Low to High. If the number is already in that range, it is unchanged. Otherwise, the closer of Low or High is returned.
Аргументы напуталь

Reply:
Или нет, надо еще раз тестануть

Reply:
Я использовал это, если будет введено неверная частота, она автоматически переключится на минимальную:
```
#define Clamp(value, low, high) 	(value <= low ? low : (value >= high ? high : value))
```